.github/workflows/test-prs.yml

```diff
@@ -291,8 +285,8 @@ jobs:
       - name: Checkout Amplify UI
         uses: actions/checkout@v2
         with:
-          ref: ${{ github.event_name == 'push' && github.ref || github.event.pull_request.head.ref }}
-          repository: ${{github.event.pull_request.head.repo.full_name}}
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
Why do we no longer need the conditionals here ?

Reply:
`event_name` is the name of the event that triggers workflow. Previously, this could be either `pull_request_target` for PRs, and `push` for push to `main`.

With this PR splitting those up and simplifying trigger like so:

```diff
on:
-  push:
-    branches: [main, ui-svelte/main]

  pull_request_target:
    branches: [main, ui-svelte/main]
    types: [opened, synchronize, labeled]
```

`github.event.event_name` will always be `pull_request_target`.

Reply:
👍